src/app/gui/maskwidget.h

```diff
@@ -0,0 +1,17 @@
+#ifndef MASKWIDGET_H
+#define MASKWIDGET_H
+#include <QWidget>
+
+/*!
+  \file    cameracalibwidget.h
+  \brief   A GUI for setting up a mask on the image
+  \author  Rui Silva, (C) 2017
+*/
+class MaskWidget : public QWidget {
```

Comment:
This class does not seem to be actually used anywhere, or did I miss something?

Reply:
good catch, forgot to remove it.